tests/test_client_parameters.py

```diff
@@ -29,6 +30,21 @@
             AwsClientParameters(api_version="1.0.0"),
             {"api_version": "1.0.0"},
         ),
+        pytest.param(
+            AwsClientParameters(
+                config=Config(
+                    region_name="eu_west_1",
+                    retries={"max_attempts": 10, "mode": "standard"}
+                )
+            ),
+            {
+                "config": {
+                    "region_name": "eu_west_1",
+                    "retries": {"max_attempts": 10, "mode": "standard"},
+                },
+            },
+            marks=pytest.mark.xfail(reason="boto3 Config not directly equivalent to a dict"),
```

Comment:
Maybe instead of xfail, we can try put this in a separate test to test equality of botocore.client.Config

Reply:
done